### ISSUE TITLE
uc04-system-tables tries running against the wrong catalog.

### ISSUE THREAD

**Issue by nova-jj** [databricks-demos/dbdemos#91]
![image](https://github.com/databricks-demos/dbdemos/assets/147086528/7c3755c5-10fb-4f94-8cba-48de71824dd9)

Install worked fine as a user against a cluster in Shared Mode.  

First, Running the 02 notebook mashed our target schema permissions, setting ownership to account users.

I reset that and made sure the service account meant to run the daily forecaster had the necessary permissions, but then last night it ran against the wrong catalog.

I've had nothing but difficulty moving this demo into a production-ready state and it appears buggy.  Your mileage may vary but these built-in-assumptions of Main Catalog are problematic to operationalize these demos.  

Might I suggest making catalog and schema inputs **required** inputs, and have processes fail when they can't touch them for whatever reason, instead of scaffolding new objects in the wrong catalog/database?

Ultimately, it's unclear to me why this is targeting Main and not the catalog I'm specifying.  I've even changed the values in the _resources/00-setup notebook, no avail:
![image](https://github.com/databricks-demos/dbdemos/assets/147086528/c37b93ea-914e-48fe-aaca-ef1a1a678461)

**Comment by nova-jj** [comment]
TLDR: Resolution: Hard Coding Job Parameter values in the Job Cluster was required to overcome this issue.

**Comment by QuentinAmbard** [comment]
hey sorry about that,  it wasn't intended to work like that, account users should apply only for dbdemos catalog.
I'm releasing a fix so that it doesn't happen again for this demo

**Comment by nova-jj** [comment]
Curious why permissions are touched _at all_.  If pointing at a pre-existing schema, why would ownership need to change?

Unity Catalog presents a unique challenge for data governance and lacks the ability to provide "deny" rules, adding complexity to traditional DBA operations; it would behoove dbdemos not to touch permissions EVER, and force users to work through adequate access as required, but that's my two cents.

**Comment by QuentinAmbard** [comment]
it's something we designed originally for internal usage, we're all super admin in some internal env, so we forced it to account users so that everybody can use it. Not something you want to do in a real deployment for sure.
